### PR TITLE
Fix MULTISITE handling during install

### PR DESCRIPTION
**Pull request by roborourke**
There are 2 additions here:

- prevent web access to the install screen
- define a constant if we detect the cli command is for the initial install

Relies on an update to CMS Installer.

### CODE REVIEW

File: inc/cli/namespace.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Altis\CMS\CLI;
+
+use WP_CLI;
+
+function bootstrap() {
```
Comment (rmccue) [inline inc/cli/namespace.php]: phpdoc? :)

File: inc/namespace.php [hunk 2]
```diff
@@ -222,3 +226,17 @@ function hide_welcome_panel( $value, int $user_id, string $meta_key ) {
...
+ */
+function prevent_web_install() {
+	if ( $_SERVER['REQUEST_URI'] !== '/wp-admin/install.php' ) {
+		return;
+	}
+
+	// Return 200 status for healthcheck.
+	status_header( 200 );
+	echo 'This site is currently unavailable';
+	exit;
```
Comment (rmccue) [inline inc/namespace.php]: This seems familiar somehow 🤔

File: inc/namespace.php [hunk 3]
```diff
@@ -222,3 +226,17 @@ function hide_welcome_panel( $value, int $user_id, string $meta_key ) {
...
+/**
+ * Prevent direct web access to wp-admin/install.php.
+ */
+function prevent_web_install() {
+	if ( $_SERVER['REQUEST_URI'] !== '/wp-admin/install.php' ) {
+		return;
+	}
+
+	// Return 200 status for healthcheck.
+	status_header( 200 );
```
Comment (rmccue) [inline inc/namespace.php]: This is actually not necessary, but let's keep it anyway so it's explicit 👍

File: inc/namespace.php [hunk 4]
```diff
@@ -222,3 +226,17 @@ function hide_welcome_panel( $value, int $user_id, string $meta_key ) {
 
 	return [ 0 ];
 }
+
+/**
+ * Prevent direct web access to wp-admin/install.php.
+ */
+function prevent_web_install() {
+	if ( $_SERVER['REQUEST_URI'] !== '/wp-admin/install.php' ) {
```
Comment (rmccue) [inline inc/namespace.php]: I guess we probably can't detect this a different way? Like, some sort of `WP_INSTALLING` constant?

Comment (roborourke) [reply inc/namespace.php]: Not that I can see

**Pull request merged**